pkg/oauthserver/auth.go

```diff
@@ -667,6 +697,33 @@ func (c *OAuthServerConfig) getAuthenticationRequestHandler() (authenticator.Req
 	return authRequestHandler, nil
 }
 
+func newIdentityUserMapperWithGroups(
+	identities userclient.IdentityInterface,
+	users userclient.UserInterface,
+	groupsInformer userinformer.GroupInformer,
+	groups userclient.GroupInterface,
+	groupsLister userlisterv1.GroupLister,
+	userIdentityMapping userclient.UserIdentityMappingInterface,
+	method identitymapper.MappingMethodType,
+) (api.UserIdentityMapper, error) {
+	userMapper, err := identitymapper.NewIdentityUserMapper(
```

Comment:
as far as i see `NewIdentityUserMapper` is unused, is there any reason to keep this separate constructor? I think we are instanciating with groups only, no?